layouts/partials/html-head.hbs

```diff
@@ -2,8 +2,8 @@
   <meta charset="utf-8">
   <title>{{#if title}}{{ title }} | {{/if}}{{ site.title }}</title>
 
-  <link rel="dns-prefetch" href="https://fonts.googleapis.com">
-  <link rel="dns-prefetch" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
IIUC, `preconnect` will create unnecessary network connections if the resource is in the browser cache, which I imagine it will be more often than not here. Correction welcome if I'm wrong about that of course.

Reply:
> IIUC, `preconnect` will create unnecessary network connections if the resource is in the browser cache, which I imagine it will be more often than not here. Correction welcome if I'm wrong about that of course.

Ref: https://stackoverflow.com/a/51059127/436641